[2023-12-11T20:17:49.683+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: s3_hook_overview.upload_to_snowflake manual__2023-12-11T20:17:28.780393+00:00 [queued]>
[2023-12-11T20:17:49.687+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: s3_hook_overview.upload_to_snowflake manual__2023-12-11T20:17:28.780393+00:00 [queued]>
[2023-12-11T20:17:49.687+0000] {taskinstance.py:1361} INFO - Starting attempt 1 of 2
[2023-12-11T20:17:49.693+0000] {taskinstance.py:1382} INFO - Executing <Task(PythonOperator): upload_to_snowflake> on 2023-12-11 20:17:28.780393+00:00
[2023-12-11T20:17:49.697+0000] {standard_task_runner.py:57} INFO - Started process 6077 to run task
[2023-12-11T20:17:49.700+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 's3_hook_overview', 'upload_to_snowflake', 'manual__2023-12-11T20:17:28.780393+00:00', '--job-id', '43', '--raw', '--subdir', 'DAGS_FOLDER/load_into_snowflake.py', '--cfg-path', '/tmp/tmpjeww3wgq']
[2023-12-11T20:17:49.702+0000] {standard_task_runner.py:85} INFO - Job 43: Subtask upload_to_snowflake
[2023-12-11T20:17:49.730+0000] {task_command.py:416} INFO - Running <TaskInstance: s3_hook_overview.upload_to_snowflake manual__2023-12-11T20:17:28.780393+00:00 [running]> on host a3992be7c3a0
[2023-12-11T20:17:49.767+0000] {taskinstance.py:1662} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='s3_hook_overview' AIRFLOW_CTX_TASK_ID='upload_to_snowflake' AIRFLOW_CTX_EXECUTION_DATE='2023-12-11T20:17:28.780393+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2023-12-11T20:17:28.780393+00:00'
[2023-12-11T20:17:49.778+0000] {copy_into_snowflake.py:139} INFO - Executing COPY command...
[2023-12-11T20:17:49.781+0000] {base.py:73} INFO - Using connection ID 'snowflake_default' for task execution.
[2023-12-11T20:17:49.781+0000] {connection.py:334} INFO - Snowflake Connector for Python Version: 3.3.1, Python Version: 3.8.18, Platform: Linux-6.5.11-linuxkit-aarch64-with-glibc2.17
[2023-12-11T20:17:49.782+0000] {connection.py:1103} INFO - This connection is in OCSP Fail Open Mode. TLS Certificates would be checked for validity and revocation status. Any other Certificate Revocation related exceptions or OCSP Responder failures would be disregarded in favor of connectivity.
[2023-12-11T20:17:49.782+0000] {connection.py:1121} INFO - Setting use_openssl_only mode to False
[2023-12-11T20:17:50.508+0000] {cursor.py:833} INFO - query: [ALTER SESSION SET autocommit=True]
[2023-12-11T20:17:50.631+0000] {cursor.py:846} INFO - query execution done
[2023-12-11T20:17:50.633+0000] {cursor.py:988} INFO - Number of results in first chunk: 1
[2023-12-11T20:17:50.635+0000] {sql.py:418} INFO - Running statement: COPY INTO my_schema.organizations
             FROM  @my_s3_stage/
        
        PATTERN='.*[.]csv'
        FILE_FORMAT=your_snowflake_file_format, parameters: None
[2023-12-11T20:17:50.636+0000] {cursor.py:833} INFO - query: [COPY INTO my_schema.organizations FROM  @my_s3_stage/  PATTERN='.*[.]csv' FILE_F...]
[2023-12-11T20:17:50.745+0000] {cursor.py:846} INFO - query execution done
[2023-12-11T20:17:50.759+0000] {connection.py:677} INFO - closed
[2023-12-11T20:17:50.819+0000] {connection.py:683} INFO - No async queries seem to be running, deleting session
[2023-12-11T20:17:50.886+0000] {taskinstance.py:1937} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/operators/python.py", line 192, in execute
    return_value = self.execute_callable()
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/operators/python.py", line 209, in execute_callable
    return self.python_callable(*self.op_args, **self.op_kwargs)
  File "/opt/airflow/dags/load_into_snowflake.py", line 36, in upload_to_snowflake
    snowflake_task.execute(context=ti.get_template_context())
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/providers/snowflake/transfers/copy_into_snowflake.py", line 140, in execute
    snowflake_hook.run(sql=sql, autocommit=self.autocommit)
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/providers/snowflake/hooks/snowflake.py", line 388, in run
    self._run_command(cur, sql_statement, parameters)
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/providers/common/sql/hooks/sql.py", line 423, in _run_command
    cur.execute(sql_statement)
  File "/home/airflow/.local/lib/python3.8/site-packages/snowflake/connector/cursor.py", line 937, in execute
    Error.errorhandler_wrapper(self.connection, self, error_class, errvalue)
  File "/home/airflow/.local/lib/python3.8/site-packages/snowflake/connector/errors.py", line 290, in errorhandler_wrapper
    handed_over = Error.hand_to_other_handler(
  File "/home/airflow/.local/lib/python3.8/site-packages/snowflake/connector/errors.py", line 345, in hand_to_other_handler
    cursor.errorhandler(connection, cursor, error_class, error_value)
  File "/home/airflow/.local/lib/python3.8/site-packages/snowflake/connector/errors.py", line 221, in default_errorhandler
    raise error_class(
snowflake.connector.errors.ProgrammingError: 002003 (02000): 01b0ec81-3200-f5b6-0000-00074442904d: SQL compilation error:
File format 'YOUR_SNOWFLAKE_FILE_FORMAT' does not exist or not authorized.
[2023-12-11T20:17:50.919+0000] {taskinstance.py:1400} INFO - Marking task as UP_FOR_RETRY. dag_id=s3_hook_overview, task_id=upload_to_snowflake, execution_date=20231211T201728, start_date=20231211T201749, end_date=20231211T201750
[2023-12-11T20:17:50.939+0000] {standard_task_runner.py:104} ERROR - Failed to execute job 43 for task upload_to_snowflake (002003 (02000): 01b0ec81-3200-f5b6-0000-00074442904d: SQL compilation error:
File format 'YOUR_SNOWFLAKE_FILE_FORMAT' does not exist or not authorized.; 6077)
[2023-12-11T20:17:50.967+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 1
[2023-12-11T20:17:50.982+0000] {taskinstance.py:2778} INFO - 0 downstream tasks scheduled from follow-on schedule check
